src/WebJobs.Script/Workers/Rpc/RpcWorkerProcess.cs

```diff
@@ -40,12 +41,14 @@ internal class RpcWorkerProcess : WorkerProcess
             _workerId = workerId;
             _serverUri = serverUri;
             _scriptRootPath = rootScriptPath;
-            _workerProcessArguments = workerProcessArguments;
+            _workerProcessArguments = rpcWorkerConfig.Arguments;
+            _workerDirectory = rpcWorkerConfig.Description.WorkerDirectory;
         }
 
         internal override Process CreateWorkerProcess()
         {
             var workerContext = new RpcWorkerContext(Guid.NewGuid().ToString(), RpcWorkerConstants.DefaultMaxMessageLengthBytes, _workerId, _workerProcessArguments, _scriptRootPath, _serverUri);
+            workerContext.EnvironmentVariables.Add(WorkerConstants.FunctionsWorkerDirectorySettingName, _workerDirectory);
```

Comment:
@amamounelsayed - This valud points to the directory of `worker.config.json` location. Is this what you were looking for?

Reply:
Thank you so much @pragnagopa for your fast response. 
Yes this is correct example from local machine bin\Debug\netcoreapp3.1\workers\java\
I can test this branch tomorrow to make sure about the value. Thank you so much again.